src/components/Search/Search.jsx

```diff
@@ -26,3 +27,12 @@ function Search(props) {
 }
 
 export default Search;
+
+Search.propTypes = {
+  initialQuery: PropTypes.string,
+  onSearch: PropTypes.func
+};
+
+Search.defaultProps = {
+  initialQuery: "What do you want?"
```

Comment:
The `initalQuery` is not correct. You should have empty or filled value.
The text "What do you want?" more looks like `placeholder`. Try to use in `input` DOM element and change value